### ISSUE TITLE
Recently added music albums seem random

### ISSUE THREAD

**Issue by steve1977** [croneter/PlexKodiConnect#793]
I am using Kodi 18.1 on Nvidia Shield TV. Applied direct paths.

Recently added music albums seem to show random albums (not the recently added ones from Plex).

Thanks for looking into this!

**Comment by croneter** [comment]
The issue is Kodi's way of deciding, when an album was actually added. SQL statement for that (table `albumview`):
```SQL
(SELECT MAX(song.dateAdded) FROM song WHERE song.idAlbum = album.idAlbum) AS dateAdded,
```
So if there is even one song that was added later to a particular album, that song's dateAdded will count for the entire album. I'm looking into changing this mechano

**Comment by croneter** [comment]
Nope, that's not the only issue. Seems the order in which PKC adds albums is relevant... :see_no_evil:

**Comment by croneter** [comment]
Fixed in PKC **BETA** version 2.10.3!! 💪 😆
